### DISCUSSION TITLE
Is it possible to change the language?

### DISCUSSION THREAD

**Discussion by Frovix** [sbtoonz/Trader_2.0#9] [Q&A] [answered]
Hello, I would like to know if it is possible to change the language of the mods? Ideally I would like to be able to do this myself via a file or editing program. Ideally this would work for most mods. Unfortunately I play with friends who speak English poorly and I would like to translate mods for them. Therefore I thought it would be useful to ask experienced modders about this. Thank you in advance for your reply

**Comment by sbtoonz** [comment, marked as answer]
> Hello, I would like to know if it is possible to change the language of the mods? Ideally I would like to be able to do this myself via a file or editing program. Ideally this would work for most mods. Unfortunately I play with friends who speak English poorly and I would like to translate mods for them. Therefore I thought it would be useful to ask experienced modders about this. Thank you in advance for your reply
> 
> 

All the item data should pull from translation that valheim gives for the item 

Then the buy and sell tabs are translated in the config file 


Let me know what stuff isn't translating and I'll work on it ?
